### ISSUE TITLE
Is UKUU no longer going to be maintained(Last commit was 4 months ago)?

### ISSUE THREAD

**Issue by caiodev** [teejee2008/ukuu#121]

**Comment by antonio-petricca** [comment]
The new UKUU release has been announced yesterday as no more free.

https://teejeetech.in/2019/01/20/ukuu-v19-01/

**Comment by gothicVI** [comment]
I'm wondering if it's gonna maintain its license and still be open source though...

**Comment by antonio-petricca** [comment]
You can see on your own the sources of the 19.1 are no more on this repo!

**Comment by gothicVI** [comment]
That's right. And eventually, that also won't happen which renders the software unnecessary for me.
Ukuu has always been a nice helper but using `wget` isn't that difficult and especially isn't worth using closed source software that requires root access and internet connection...

**Comment by gothicVI** [comment]
That's exactly what I just said.

**Comment by caiodev** [comment]
Ok, thank you guys. I'll just stick to these .deb kernel files. It's one less program on my computer anyways

**Comment by antonio-petricca** [comment]
You could use the old version of ukuu anywau.

Il Mar 22 Gen 2019, 19:44 Caio Costa <notifications@github.com> ha scritto:

**Comment by ThoreauHenry** [comment]
https://github.com/pimlie/ubuntu-mainline-kernel.sh/blob/master/README.md
